### ISSUE TITLE
End of Lessons Review Ticket

### ISSUE THREAD

**Issue by SrijDude3416** [SrijDude3416/srijan_2025#5]

**Comment by jm1021** [comment]
Missing is personal blog.   Memories and perfunctory answers don't do it for me.

**Comment by jm1021** [comment]
Blog - in association with Units and Teaching
Memory - something to recall your individual presentation
Relevancy - something impactful you learned
Beyond Perfunctory - something that shows you want to learn

**Comment by 7mwang** [comment]
I thought it was really good that you showed the communication with your team, as that's a really important part of teamwork. I also liked how you showed your cpmmits.

**Comment by eshaank1** [comment]
I like how you showed the conversation with you group showing collaboration.

**Comment by Saaras859** [comment]
Memory: primitives to monkeys nice to remember, has a nice Jon funny prank so its pretty entertaining. Going first is unique

Personal Blog: I liked that you mentioned all the lessons and how you fit all the cb requirements

Relevance: I think something impactful is how he orders everything and learns all the information

Beyond Perfunctory: Talks about how he learned how to do the hacks

**Comment by jm1021** [comment]
Blog - in association with Units and Teaching
Memory - something to recall your individual presentation
Relevancy - something impactful you learned
Beyond Perfunctory - something that shows you want to learn

Took content for tldr.  Condensed version of main topics.  Showed an image on 2D.   Chain of candy and expired fruit gummies.  2D array algorithm on other team mentioned.

**Comment by anikabhatnagar20** [comment]
I like how you organized your takeaway and how you made engaging notes, etc.

**Comment by Dabear14** [comment]
Hi Srijan, 
I liked how made a compact review of everything we learned and explained all the topics on your issue as well.

**Comment by tarunja1ks** [comment]
Blog - He did notes of the content and made a table that can expand for basic information about the unit, and also condensed versions of each topic. 
Memory - He did a groupchat memory of the candy discussion, and how we didn't have any candy. 
Relevancy - Focused on the other groups such as Anushas group as they did a different intend solution for one of the popcorn hacks. He also liked Aidan's method of teaching and demoing with the cards.
Beyond Perfunctory -  Doing notes on each lesson, and going above and beyond for revieing/remmebering lessons

**Comment by AidanLau10** [comment]
I like your presentation skills and how you use your hands to control the audience. You definitely know what you're talking about what what to improve on.

**Comment by tanav-kambhampati** [comment]
Hey Srijan, 
I'm glad that you guys learned a lot from going first on the team teach, I definitely learned a lot from it as well. Your memory is also pretty funny. I'm glad you were able to find things relevant because they will be useful in the future.

**Comment by KaydenLe** [comment]
I liked that your presentation and how you your hands a lot. Your memory was good. You definitely know what you need to do.

**Comment by hanlunli** [comment]
TLDR of TLDR is a nice touch. I like how you included the anecdote about collaborating with your team.

**Comment by beijanm** [comment]
This was a great presentation and the addition of a memory was great.

**Comment by miggysp** [comment]
Hi Srijan, 
I liked your presentation and your overall work this Sprint. The way you presented it shows how much you know. Your memory was funny.

**Comment by Imaad08** [comment]
I like your organization of the different units and your memory is a pretty funny moment.

**Comment by tanayshah1** [comment]
Going first must have been tough, yet as guinea pigs you guys did very well. Your presentation was very interesting and engaging and it was really easy to follow along with what you were saying.

**Comment by spooketti** [comment]
Blog - Did notes on content and helped organize the team
Memory - i brought jolly ranchers on presentation day though :(
Relevancy - The TLDR of  TLDR made me think of derivatives: awesome
Beyond Perfunctory - Was nice going over going first

**Comment by avanthikadaita** [comment]
Great presentation. Your team teach was very well done and I liked how you showed how much you learned and still remember.

**Comment by SGTech08** [comment]
I love how you showed how you guys communicated on your team teach so nice on jon.

**Comment by vibha-yganji** [comment]
Your presentation was very engaging to watch. I liked that you referred to what you liked about other team teaches to show your reflection on your own teach.

**Comment by alishahussain** [comment]
You have a really funny presentation, and I think you have everything. I like how your studying for the team teach associates words so that you can remember and do better. Good job!
